rosdep/base.yaml

```diff
@@ -388,6 +388,13 @@ can-utils:
   debian: [can-utils]
   fedora: [can-utils]
   ubuntu: [can-utils]
+castxml:
+  arch: [castxml]
+  debian: [castxml]
+  fedora: [castxml]
+  gentoo: [dev-cpp/castxml]
```

Comment:
The link you provided for gentoo isn't the normal place we look for gentoo packages.  Running this [search](https://packages.gentoo.org/packages/search?q=castxml) yields no results for gentoo, so I'd remove it.  The rest look OK to me.

Reply:
Done